From: sunshineinabox <aqemail@gmail.com>
Date: Tue, 29 Sep 2026 23:40:00 -0700
Subject: [PATCH] opengl: test GL_OES_standard_derivatives with defined()

GLSL ES 3.00 rejects undefined identifiers in #if, which the Mali blob
enforces.
---
--- a/src/opengl/base.frag
+++ b/src/opengl/base.frag
@@ -1,6 +1,6 @@
 #version 140
 
-#if GL_OES_standard_derivatives && TRAIT_ROUNDED_CORNERS && TRAIT_BORDER
+#if defined(GL_OES_standard_derivatives) && TRAIT_ROUNDED_CORNERS && TRAIT_BORDER
 #extension GL_OES_standard_derivatives : enable
 #endif
 
